src/amo/pages/LanguageTools/index.js

```diff
@@ -55,9 +56,7 @@ const sortedLocales = sortedLanguages.map((language) => language.locale);
 export const LanguageToolList = ({
   languageTools,
 }: LanguageToolListProps): null | React.Node => {
-  if (!languageTools || !languageTools.length) {
-    return null;
-  }
+  invariant(languageTools, 'languageTools is required');
```

Comment:
My suggestion was to check that `languageTools.length` was not falsey (or greater than 0), as that is part of what was being checked before. Flow should protect us from `languageTools` itself being falsey.